.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug Report
+about: Describe an iRODS bug
+title: ''
+labels: ''
```

Comment:
Noticed that the enhancement template has an enhancement label whereas this does not have any labels. Is that on purpose?

Reply:
yes, i think we like to determine/verify if things are actually bugs, before adding the label.

enhancements are... well, safer to label early i think.

Reply:
Okay. I think we're going to end up touching the labels in any case as we triage things, so I could see it either way. Up to you

Reply:
yeah, happy to ship this first with zero auto labeling.   will update the feature request one.